ReactNative/ReactTwitter/core/service/twitter-requests-service.js

```diff
@@ -0,0 +1,51 @@
+var OauthHelper = require('../service/oauth-helper.js')
+
+const CONSUMER_KEY = 'aqPSTs1FT2ndP7qXi247BtbXd'
```

Comment:
I'd load these from a `config.json`, it's easy as doing `const config = require('path/to/config.json')`